lib/src/regular_expressions/regular_expressions_extensions.dart

```diff
@@ -35,24 +35,24 @@ class RegExpComposer {
     // print(result);
     // print("");
 
-    index = 0;
-    result = _replace(result, _matchPositiveLookbehind, (Match m) {
-      final replacedString = "(?<plb$_groupNameIndexSep$index>";
-      index += 1;
+    // index = 0;
```

Comment:
Any idea why we needed to comment this out? I assume it was there for a reason. Any idea how it's supposed to work, if it's currently broken?